### DISCUSSION TITLE
Does this extension sync files from the file library, including from providers like Amazon S3 or DigitalOcean Spaces?

### DISCUSSION THREAD

**Discussion by gridsystem** [tractr/directus-sync#107] [Q&A]
I've had a look through the code and the repo, and see a reference to a `folders` collection, but can't see for sure if that involves syncing files too.

I'd like to download all of the files and run them in a local copy of Directus, so that I have an entirely self-contained CMS/API, that doesn't reference any external files, so that if the online version is ever taken offline, I still have a working backup.

**Comment by EdouardDem** [comment]
Hi @gridsystem 
The sync of the collection folders is only the declaration of the folder, i.e. the entry in `directus_folders` table.
Sync of custom data and files is not supported yet, See #1
